sike/sikep751/sikep751.go

```diff
@@ -0,0 +1,168 @@
+// This file contains APIs for Supersingular Isogeny Key Encapsulation (SIKE) protocol
+// which is recently submitted to NIST PQC standardization workshop.
+// The underlying core functions are based on SIDH API developed by Cloudflare.
+// SIKE requires NIST's approved hash functions such as sha3 and cSHAKE256 to encapsulate
+// the key based on NIST specifications. The sha3 package along with cSHAKE256 implementation
+// is placed inside the p751sidh package. Note that the standard sha3 package from
+// "golang.org/x/crypto/sha3" does not include cSHAKE256 functions.
+//
+// Author: Amir Jalali  				ajalali2016@fau.edu
+// Date: Feb 2018
+package sikep751
+
+import (
+	"bytes"
+	"io"
+)
+
+import . "github.com/cloudflare/p751sidh/sha3"
+
+import . "github.com/cloudflare/p751sidh/sidh/sidhp751"
```

Comment:
It won't work as "sidh" directory doesn't exist. In fact the import should stay as it is, but directory structure should be reorganized
This would be better if imports are in one block:
```
import (
	"bytes"
	"io"

	. "github.com/cloudflare/p751sidh/sha3"
	. "github.com/cloudflare/p751sidh/sidh/sidhp751"
)
```